docs/specs/sequences/endpoint_reconfiguration.md

```diff
@@ -22,11 +22,11 @@ sequenceDiagram
     participant D as Device
     participant E as Original Endpoint
     participant E' as New Endpoint
-    E->>D:CONFIG MESSAGE<br>blobset.blobs._iot_endpoint_config.base64 = <ENDPOINT><br>blobset.blobs._iot_endpoint.blob.phase = "final"
-    D->>E:STATE MESSAGE<BR>blobset.blobs._iot_endpoint_config.blob.phase = "apply"
+    E->>D:CONFIG MESSAGE<br>blobset.blobs._iot_endpoint_config.base64 = <ENDPOINT><br>blobset.blobs._iot_endpoint.phase = "final"
+    D->>E:STATE MESSAGE<BR>blobset.blobs._iot_endpoint_config.phase = "apply"
     D-->>E':CONNECTION ATTEMPT
     E'->>D:CONFIG MESSAGE
-    D->>E':STATE MESSAGE<BR>blobset.blobs._iot_endpoint_config.blob.phase = "final"
+    D->>E':STATE MESSAGE<BR>blobset.blobs._iot_endpoint_config.phase = "final"
```

Comment:
I think we should add an additional config/state sequence at the end that starts with `config.blobset.blobs._iot_endpoint_config = null` and then we verify that the `state.blobset.blobs._iot_endpoint_config == null`  The fundamental reason is that we want to verify that the state block only shows up when expected.